docs/src/content/docs/reference/configuration.md

```diff
@@ -379,3 +379,29 @@ By default, this feature relies on your repository’s Git history and may not b
 Define if the footer should include previous and next page links.
 
 A page can override this setting or the link text and/or URL using the [`prev`](/reference/frontmatter/#prev) and [`next`](/reference/frontmatter/#next) frontmatter fields.
+
+### `favicon`
+
+**type:** `string`  
+**default:** `'/favicon.svg'`
+
+Set the path of the default favicon for your website.
```

Comment:
Do you want to specify the accepted file extensions here?

Reply:
Hmm, that is a good point. I wonder tho if we should explicitly list them or link to a reference of some sort, e.g. this one https://en.wikipedia.org/wiki/Favicon#Image_file_format_support ?